tests/snmpsim/ceraos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
```

Comment:
Sorry @barryodonovan I thought you were asking about sysObjectID and not SysDescr. Discovery is based on sysObjectID (1.3.6.1.2.1.1.2.0) so as long as that one returns something with .1.3.6.1.4.1.2281 on it (which is CeraOS i think) it will discover properly!